examples/nimble_heart_rate_sensor/Makefile

```diff
@@ -0,0 +1,32 @@
+# name of your application
+APPLICATION = nimble_heart_rate_sensor
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= nrf52dk
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Some RIOT modules needed for this example
+USEMODULE += event_timeout
+
+# Include NimBLE
+USEPKG += nimble
+USEMODULE += nimble_svc_gap
+USEMODULE += nimble_svc_gatt
+
+# We also use the AD part of the BLE helper module
+USEMODULE += bluetil_ad
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# REMOVE
+CFLAGS += -DDEBUG_ASSERT_VERBOSE
```

Comment:
I guess this should be removed before merging

Reply:
yes, missed that one. Good thing I did put the comment to it :-)